### ISSUE TITLE
FR: Show when device is charging

### ISSUE THREAD

**Issue by CZonin** [andyvorld/LGSTrayBattery#73]
Would it be possible to show an indicator on the tray icon when the device is charging?

**Comment by andyvorld** [comment]
Pre-release version 3.0 has this feature
https://github.com/andyvorld/LGSTrayBattery/releases/tag/v3.0.0-rc.1

**Comment by andyvorld** [comment]
Please use https://github.com/andyvorld/LGSTrayBattery/releases/tag/v3.0.0-rc.2, instead.
Had a packaging bug that caused the native manager to not run.

**Comment by CZonin** [comment]
Awesome! I'll give it a try when I get home later.

**Comment by CZonin** [comment]
Just tested it with my G502 X Plus. Battery displays properly with icon and numeric, but it charging status doesn't seem to show.

**Comment by realgooseman** [comment]
I tested version 3.0.0-rc.2 with my G703 mouse and G915 keyboard.

The switch to the "charging" icon indicator in tray menu is not automatic. When devices are plugged in you have to hit the "Rediscover Devices" button to trigger the switch to charging status (or inversely, back to battery percentage). So it needs to be automated instead of having to manually trigger it, for smoother experience.

**Comment by CZonin** [comment]
Ah okay got it that makes sense. I have a small charging pad that I charge my mouse on so can't really move it to refresh the status.

**Comment by andyvorld** [comment]
Yeah, due to the implementation with the native manager, the device is polled every 10min on succesful reads. However wire/wireless devices will cause a device arrival notification which instantly updates the icon. If you are using the GHUB manager on the other hand caches your battery status for a few seconds before updating.

I might make it an option to change the polling rate so that it can be more or less frequent.
